iOS/Shared/ModelApplication.swift

```diff
@@ -55,14 +55,14 @@ class ModelApplication: ITMApplication {
         viewController.itmNativeUI?.addComponent(DocumentPicker(viewController: viewController, itmMessenger: ITMViewController.application.itmMessenger))
     }
       
-    override func getUrlHashParams() -> String {
-        var hashParams = ""
+    override func getUrlHashParams() -> HashParams {
```

Comment:
This change is dependent on the iTwin/mobile-sdk-ios PR. Try to merge the two PRs as closely together as possible.

Reply:
Ready.